pkg/reconcilermanager/controllers/rootsync_controller_manager_test.go

```diff
@@ -563,6 +563,9 @@ func startControllerManager(ctx context.Context, t *testing.T, fakeClient *synce
 		MapperProvider: func(_ *rest.Config) (meta.RESTMapper, error) {
 			return fakeClient.RESTMapper(), nil
 		},
+		// The underlying library uses a fixed port for serving metrics, which can
```

Comment:
is this a new bug you found?

Reply:
Yeah this popped up more frequently after adding the resource-group unit tests. I think it was a pre-existing bug but the addition of new unit tests made it fail more often